Makefile

```diff
@@ -18,30 +22,37 @@ test-views-v1.1:
 	cd dashboard/v1.1/ && yarn run build
 	cd dashboard/v1.1/ && yarn start &
 
+## Removes all residual files.
 clean:
 	rm -R build/ bench-routes
 
+## Runs Golang unit tests
 test: build
 	go clean -testcache
 	go test -v ./...
 
+## Runs Golang unit tests
```

Comment:
@chetanyakan Can you please update the comment here?

Reply:
Done.